hatchways/team-fuji
# Feature sending images #26

### What this PR does:
- Allows users to send Images between each other

### Screenshots / Videos:
![Screen Shot 2021-06-09 at 9 28 16 PM](https://user-images.githubusercontent.com/58789967/121450436-baf06d80-c969-11eb-8aa8-1be7557a9d91.png)
![Screen Shot 2021-06-09 at 9 28 43 PM](https://user-images.githubusercontent.com/58789967/121450440-bcba3100-c969-11eb-9949-78af8639da9a.png)
![Screen Shot 2021-06-09 at 9 28 56 PM](https://user-images.githubusercontent.com/58789967/121450446-be83f480-c969-11eb-9a33-cdbf9a25bea2.png)
"User Sending the Image is on the left"


### Any information needed to test this feature:
  1. Click the Attach Image Icon in the Input Bar
  2. Select and Submit as many images as desired
  3. Once images have loaded into the input bar, click the input bar and hit "Enter" key
  4. Open another browser, login into another account and see message Sent to another user
